storage/inchi/src/test/java/org/openscience/cdk/inchi/InChIToStructureTest.java

```diff
@@ -232,4 +232,14 @@ public void s_penta_2_3_diene() throws Exception {
         assertThat(element.focus(), is(container.getAtom(4)));
         assertThat(element.winding(), is(ITetrahedralChirality.Stereo.CLOCKWISE));
     }
+
```

Comment:
Ah. Good catch. Yes, that patch was already in master, but it seems I missed that it kept this bit. Sorry.